specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1955 @@
+            "get": {
+                "tags": ["DiscoveredSecuritySolutions"],
+                "description": "Gets a list of discovered Security Solutions for the subscription and location.",
+                "operationId": "DiscoveredSecuritySolutionsSubscriptionLocation_List",
+                "parameters": [{
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/AscLocation"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/DiscoveredSecuritySolutionList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/discoveredSecuritySolutions/{discoveredSecuritySolutionName}": {
+            "get": {
+                "tags": ["DiscoveredSecuritySolutions"],
+                "description": "Gets a specific discovered Security Solution.",
+                "operationId": "DiscoveredSecuritySolutionsSubscriptionLocation_Get",
+                "parameters": [{
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "$ref": "#/parameters/AscLocation"
+                    },
+                    {
+                        "$ref": "#/parameters/DiscoveredSecuritySolutionName"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/DiscoveredSecuritySolution"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/jitNetworkAccessPolicies": {

```

Comment:
Just make sure you are doing the linked access check on the virtual machines when doing the PUT.

Reply:
already applied in the manifest